apps/internal-ui/src/components/Alert/AlertBox.tsx

```diff
@@ -0,0 +1,35 @@
+import clsx from 'clsx';
+import { PropsWithChildren } from 'react';
+
+import { ALERT_THEME_STYLES } from '@/components/Alert/constants';
+import { useAlertContext } from '@/components/Alert/context';
+import { AlertBoxProps } from '@/components/Alert/types';
+
+const AlertBox = ({
+  children,
+  className,
+}: PropsWithChildren<AlertBoxProps>) => {
+  const { theme, useTitle, collapsible, collapsed } = useAlertContext();
+  const useCollapse = useTitle && collapsible;
+
+  return (
+    <div
+      role='alert'
+      className={clsx(
+        className,
+        'in-flex-v-stack rounded-in-8 items-start gap-x-1.5 break-keep px-4',
```

Comment:
p3: gap-x는 필요 없지 않을까요?

```suggestion
        'in-flex-v-stack rounded-in-8 items-start break-keep px-4',
```